src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -195,7 +195,7 @@ class editIndex extends viewModelBase {
                         });
                     return canActivateResult;
                 } else {
-                    this.editedIndex(indexDefinition.empty());
+                    this.editedIndex(indexDefinition.empty(this.defaultDeploymentMode()));
```

Comment:
like we talked I think we have race here.

Reply:
removed